dynamics/src/include/BrittleCGDynamicsKernel.hpp

```diff
@@ -163,6 +163,22 @@ template <int DGadvection> class BrittleCGDynamicsKernel : public CGDynamicsKern
         }
     }
 
+    double getIceOceanStressElement(const std::string& name, const int i) const override
+    {
+        const double FOcean = params.COcean * params.rhoOcean;
+
+        const double uOceanRel = uOcean(i) - avgU(i);
+        const double vOceanRel = vOcean(i) - avgV(i);
+        const double cPrime = FOcean * std::hypot(uOceanRel, vOceanRel);
+
+        if (name == uIOStressName)
+            return cPrime * (uOceanRel * cosOceanAngle - vOceanRel * sinOceanAngle);
+        else if (name == vIOStressName)
+            return cPrime * (vOceanRel * cosOceanAngle + uOceanRel * sinOceanAngle);
+        else
+            return std::nan("");
```

Comment:
Perhaps `std::numeric_limits<double>::quiet_NaN()`. Creating a NaN with a string is just bizarre looking. Standard and well defined, yes, but just weird.

Reply:
I didn't know about ``std::numeric_limits<double>::quiet_NaN()``. I'll use that because I agree about the string bit o_o It's a bit of a verbose solution, but it's fine.